src/schema.jl

```diff
@@ -36,6 +36,37 @@ function create_dataset_schema(df, name, description)
     json(schema)
 end
 
+## create csv structure for dataset to be sent to Domo.
+function create_csv_structure(df)
+    csv_data = ""
+
+    for row in eachrow(df), col_num in 1:ncol(df)
+        if col_num < ncol(df) && rownumber(row) < nrow(df)
+            csv_data = csv_data * string(
+                if ismissing(row[col_num]) "" else row[col_num] end
```

Comment:
There's probably a cleaner version to do this.

Reply:
And it's also probably a good place to convert something like a boolean to Domo's format. I'm actually not even sure Domo has a "true" sense of a boolean so it probably just needs to be a string.